spec/ruby/core/tracepoint/inspect_spec.rb

```diff
@@ -98,7 +98,7 @@ class TracePointSpec::C
     TracePoint.new(:thread_begin) { |tp|
       next unless Thread.current == thread
       inspect ||= tp.inspect
-    }.enable do
+    }.enable(target_thread: nil) do
```

Comment:
That's needed I guess because the `thread_begin`/`thread_end` are triggered inside the new thread and not in the parent thread, right?

Reply:
Correct.